lib/rucio/common/policy.py

```diff
@@ -34,7 +34,11 @@ def get_policy() -> str:
         try:
             policy = config_get('policy', 'permission')
         except (NoOptionError, NoSectionError):
-            policy = 'atlas'
+            try:
+                policy = config_get('permission', 'policy')
+            except (NoOptionError, NoSectionError):
+                policy = 'def'
```

Comment:
It could be worth adding a warning log here to mention that the policy wasn't found and we are falling back to `def`

Reply:
I think that's a good idea. I'll add the warning.